### PR TITLE
{bio}[GCC/10.2.0] Metal v2020.05.05

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso

**Comment by robert-mijakovic** [comment]
Test report by @robert-mijakovic
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login01.meluxina.lxp.lu - Linux RHEL 8.3, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/8bb6f2536bd456824def34452d0d560f for a full test report.

File: easybuild/easyconfigs/m/Metal/Metal-2020-05-05-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'Metal'
+version = '2020-05-05'
+
+homepage = 'http://csg.sph.umich.edu/abecasis/Metal/'
+description = """Metal - Meta Analysis Helper. The METAL software is designed to facilitate meta-analysis of large
+ datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```
Comment (verdurin) [inline easybuild/easyconfigs/m/Metal/Metal-2020-05-05-foss-2020b.eb:10]: @fizwit any thoughts on whether this could be moved down to `GCC`?

Comment (fizwit) [reply easybuild/easyconfigs/m/Metal/Metal-2020-05-05-foss-2020b.eb:10]: compiles and runs with GCC 10.2.0, updating PR

Comment (Micket) [reply easybuild/easyconfigs/m/Metal/Metal-2020-05-05-foss-2020b.eb:10]: You missed removing the foss variant, so now there are 2 easyconfigs in this PR. `eb --update-pr 13485 :Metal-2020-05-05-foss-2020b.eb` (b.t.w I recommend that you don't specify `--pr-title`  on new PR's  unless you are only changing something

**Review by Micket — DISMISSED** [review]

**Review by lexming — CHANGES_REQUESTED** [review]
@fizwit Thanks for the update. I think that we can avoid the `postinstallcmds` by using the easyblock `CMakeMakeCp`. If you agree, I made a PR with this change in https://github.com/fizwit/easybuild-easyconfigs/pull/9 . It simplifies this easyconfig quite a bit.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b6e08b8efc51d892b080d12af404633b for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @fizwit !

**Pull request merged**